_docs-2/administration/caching.md

```diff
@@ -18,6 +18,10 @@ for tables where read performance is critical.
 
 ## Configuration
 
+The {% plink tserver.cache.manager.class %} property controls which block cache implementation is used within the tablet server. Users
+can supply their own implementation and set custom configuration properties to control it's behavior (see org.apache.accumulo.core.spi.cache.BlockCacheManager$Configuration.java).
+An example off-heap block cache implementation and configuration can be found at: https://github.com/keith-turner/accumulo-ohc.
+
```

Comment:
An example would serve great as a block post, but I don't think it's particularly helpful as a mere link to one of many possible external implementations.

Reply:
> An example would serve great as a block post

Did you mean adding a code block to caching.md, or a blog post?

Reply:
I meant blog post. Sorry for the typo.